ENG4000-Team-A/capstone-project
# Timer Page functionality & SSL Setup

**Timer Page**
- Fixed Flickering Issue

To test (WINDOWS - Command Prompt):
1. Open a terminal in /frontend directory
2. run "set HTTPS=true", followed by "npm start"
3. The browser should pop-up with a "This site is unsafe warning"
4. Run the ssl_setup.bat script. This creates a cert/encrypted key for localhost
5. The homepage should pop up. There should be a lock icon beside the browser
![image](https://user-images.githubusercontent.com/33265426/154690145-6f4d8915-332f-485a-8480-6979be456839.png)

-------
Django changes:
URLs:
/timer/?uname (GET: returns timer for user. POST: stops timer, need to pass action = "stop" in body )
/start_timer/id (id= machine id. POST only. need to pass uname (username) in body)

Model: User_uses_Model. New field init_Balance
tasks.py: Updated stop time function to timer specification.

-------
Fixed MachineInfo.js
